backend/app/services/subscription/service.py

```
@@ -0,0 +1,778 @@
+        self,
+        db: Session,
+        *,
+        webhook_token: str,
+    ) -> Optional[Kind]:
+        """Get a Subscription by webhook token."""
+        # Query all active subscriptions and filter by webhook token
+        subscriptions = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "Subscription",
+                Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)